priv/package.json

```diff
@@ -36,7 +36,14 @@
     "mocha": "3.2.0",
     "style-loader": "0.13.0",
     "url-loader": "0.5.7",
-    "webpack": "1.12.9"
+    "webpack": "1.14.0",
+    "react-bootstrap": "0.30.7",
```

Comment:
I thought about it. Out of curiosity what is the difference between this 2 types of deps. All in all everything is digested by webpack and we don't have node running. The only reason I can see is clarity, is there something else?